metricbeat/module/kubernetes/state_deployment/state_deployment_test.go

```diff
@@ -132,5 +123,23 @@ func testCases() map[string]map[string]interface{} {
 			"replicas.unavailable": 0,
 			"replicas.updated":     1,
 		},
+		"jenkins@wise-lynx-jenkins": {
```

Comment:
I also had to add this two "expected" events that were missing when running tests. I don't understand how they weren't failing before :thinking:

Reply:
The fields exist in the test file so I assume it's correct 🤞